documentation/modules/ROOT/partials/modules/mysql-connector/con-mysql-connector-events.adoc

```diff
@@ -371,28 +457,66 @@ The value of an _update_ change event on the `customers` table has the exact sam
       "query": "UPDATE customers SET first_name='Anne Marie' WHERE id=1004"
     },
     "op": "u", // <4>
-    "ts_ms": 1465581029523 // <5>
+    "ts_ms": 1465581029523 
```

Comment:
@gunnarmorling  I just noticed that there are two "ts_ms" fields. One is nested in the "source" field and the other is a sibling of the "source" field. The sibling instance has more precision. Should the doc say something about this? And if so, what should the doc say?

Reply:
@gunnarmorling - please let me know if you want to add something to the doc about this.

Reply:
Hm, I'm thinking that might be a typo because I'm pretty sure the timestamp field in the `source` block should be `ts_sec` while the sibling is `ts_ms`.  Perhaps just fixing the field name itself would be sufficient to give the explanation for the precision difference.

Reply:
Thanks. I changed the instances inside the source block to "ts_sec", and also the instance in the schema description that corresponds to that. I think I got it right :-)